### PR TITLE
WRR-10054: Update package dependencies to fix security vulnerabilities

### PR DESCRIPTION
**Pull request by alexandrumorariu**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Updated minor dependencies for agate

agate @enact/ui-test-utils ^1.0.9 → ^1.0.10
agate/samples/ @reduxjs/toolkit ^2.2.7 → ^2.3.0
agate/samples/sampler babel-loader ^9.1.3 → ^9.2.1

No breaking changes were found

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRR-10054

### CODE REVIEW

File: samples/sampler/package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "@storybook/react": "^8.2.9",
     "@storybook/react-webpack5": "^8.2.9",
     "@storybook/theming": "^8.2.9",
-    "babel-loader": "^9.1.3",
+    "babel-loader": "^9.2.1",
```
Comment (daniel-stoian-lgp) [inline samples/sampler/package.json:36]: please also update "@enact/storybook-utils": "^6.0.0-rc.2",

Comment (alexandrumorariu) [reply samples/sampler/package.json:36]: Done

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

**Review by hong6316 — APPROVED** [review]
👍

**Pull request merged**